### PR TITLE
feat: Allow to upload more filetypes. Support highlighting.

### PR DESCRIPTION
**Pull request by yammesicka**

### CODE REVIEW

File: lms/templates/view.html [hunk 1]
```diff
@@ -39,7 +39,7 @@ <h1>תצוגת תרגיל {{ solution['exercise']['id'] }}: {{ solution['exercis
   <div id="view-body">
     <div class="code-view-container">
       <div id="code-view" data-id="{{ solution['id'] }}" data-file="{{ current_file.id }}" data-exercise="{{ solution['exercise']['id'] }}" data-role="{{ role }}">
-        <pre><code id="python-user-code" class="language-python line-numbers highlight">{{- current_file.code | trim(chars=' ') | e -}}</code></pre>
```
Comment (orronai) [inline lms/templates/view.html:42]: In my.js file there is we get this element by id of python-user-code.
Need to change there also the id.

File: lms/static/prism.css [hunk 2]
```diff
@@ -173,15 +168,15 @@ pre[data-line] {
...
   position: absolute;
-  top: 0.4em;
-  left: 0.6em;
+  top: .4em;
+  left: .6em;
   min-width: 1em;
-  padding: 0 0.5em;
-  background-color: hsla(24, 20%, 50%, 0.4);
+  padding: 0 .5em;
+  background-color: hsla(24, 20%, 50%,.4);
```
Comment (orronai) [inline lms/static/prism.css:175]: Just styling: no space between the last comma and the .4

File: lms/utils/files.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    'h': 'c',
+    'htm': 'html',
+    'js': 'javascript',
+    'md': 'markup',
+    'ps1': 'powershell',
+    'psm1': 'powershell',
+    'py': 'python',
+    'rb': 'ruby',
+    'sh': 'bash',
+    'sh': 'bash',
```
Comment (orronai) [inline lms/utils/files.py:12]: sh is written twice

File: lms/templates/view.html (hunk 1 above)
Comment (yammesicka) [reply lms/templates/view.html:42]: Awesome catch, 10x

**Review by gal432 — APPROVED** [review]

**Pull request merged**